http-client/src/main/java/io/airlift/http/client/HttpClientModule.java

```diff
@@ -153,106 +137,11 @@ public HttpClient get()
                     .addAll(injector.getInstance(Key.get(new TypeLiteral<Set<HttpRequestFilter>>() {}, annotation)))
                     .build();
 
-            JettyIoPoolManager ioPoolProvider;
-            if (injector.getExistingBinding(Key.get(JettyIoPoolManager.class, annotation)) != null) {
-                log.debug("HttpClient %s uses private IO thread pool", name);
-                ioPoolProvider = injector.getInstance(Key.get(JettyIoPoolManager.class, annotation));
-            }
-            else {
-                log.debug("HttpClient %s uses shared IO thread pool", name);
-                ioPoolProvider = injector.getInstance(JettyIoPoolManager.class);
-            }
-
+            verify(injector.getExistingBinding(Key.get(JettyIoPoolManager.class, annotation)) != null, "binding should exist");
```

Comment:
This check isn't needed anymore. We know it exists because we bound it at the same time we created this provider.